corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -224,6 +215,11 @@ FormplayerFrontend.module("Menus.Views", function (Views, FormplayerFrontend, Ba
 
     Views.CaseTileView = Views.CaseView.extend({
         template: "#case-tile-view-item-template",
+        templateHelpers: function () {
+            var dict = Views.CaseTileView.__super__.templateHelpers.apply(this, arguments);
```

Comment:
huh haven't seen `__super__` used before, do you know if it's well supported in browsers?